addons/website_sale/static/src/interactions/tracking.js

```diff
@@ -0,0 +1,99 @@
+import { Interaction } from '@web/public/interaction';
+import { registry } from '@web/core/registry';
+
+export class Tracking extends Interaction {
+    static selector = '.oe_website_sale';
+    dynamicContent = {
+        'form a.a-submit': { 't-on-click': this.onAddProductToCart },
+        'a[href^="/shop/checkout"]': { 't-on-click': this.onCheckoutStart },
+        'a[href^="/web/login?redirect"][href*="/shop/checkout"]': {
+            't-on-click': this.onCustomerSignin,
+        },
+        'a[href="/shop/confirm_order"]': { 't-on-click': this.onOrder },
+        'button[name="o_payment_submit_button"]': { 't-on-click': this.onOrderPayment },
+        _root: {
+            't-on-view_item_event': this.onViewItem,
```

Comment:
fix: `(ev) => this.onViewItem(ev)`, same for next line: `(ev) => this.onAddToCart(ev)`